backend/molgenis-emx2-webapi/src/main/java/org/molgenis/emx2/web/StaticFileMapper.java

```diff
@@ -90,6 +90,9 @@ public static void addFileToContext(Context ctx, String path, String mimeType) {
       }
       if (mimeType == null) {
         mimeType = Files.probeContentType(Path.of(path));
+        if (mimeType.equals("text/plain") && path.endsWith(".js")) {
+          mimeType = "text/javascript";
+        }
```

Comment:
In the first version I used URLConnection.guessContentTypeFromName(path)

Maybe we could combine them
mimeType = URLConnection.guessContentTypeFromName(path);
if null:
   mimeType = Files.probeContentType(path)
if null:
   mimeType = "application/octet-stream"
?

Could you test if that works? Else this fix is fine

Reply:
Also works! :)